esp/templates/users/profiles/usercontact.html

```diff
@@ -74,7 +74,14 @@ <h4>Address:</h4>
 
 {{ form.address_postal }}
 
-<h4>Phone number{% if form.phone_day and form.phone_cell %}s{% endif %}:<font color="red">*</font></h4>
+<h4>Phone number{% if form.phone_day and form.phone_cell %}s{% endif %}:{% if profiletype != "student" or require_student_phonenum %}<font color="red">*</font>{% endif %}</h4>
```

Comment:
I think we only want the red asterisk if a) the user is a teacher or b) the user is a student and `require_student_phonenum` is true, to match the behavior here:
https://github.com/learning-unlimited/ESP-Website/blob/3fbdf8d0e1eb63f04f6bcd28505e8254569d90fa/esp/esp/users/forms/user_profile.py#L72-L74